pynetbox/models/dcim.py

```diff
@@ -119,6 +119,23 @@ def units(self):
         """
         return RODetailEndpoint(self, "units", custom_return=RUs)
 
+    @property
+    def elevation(self):
+        """ Represents the ``elevation`` detail endpoint.
+
+        Returns a DetailEndpoint object that is the interface for
+        viewing response from the elevation endpoint updated in Netbox version 2.8.
+
+        :returns: :py:class:`.DetailEndpoint`
+
+        :Examples:
+
+        >>> rack = nb.dcim.racks.get(123)
+        >>> rack.elevation.list()
+        {"get_facts": {"interface_list": ["ge-0/0/0"]}}
```

Comment:
Quick nit (I just realized I have a typo in the `units()` docs too :man_facepalming:), but could you add a quick sample output here? I don't have a 2.8 instance readily available.